### ISSUE TITLE
Table A11y updates

### ISSUE THREAD

**Issue by StephDriver** [openlibhums/janeway#4055]
- [x] review use of tables across platform

From #4194 the following issues have been identified:
- Tables do not consistently have `<th>` tags for both rows and columns.  This occurs in both tables that are part of the main template for the page, e.g. article dates shown on the material theme, and for tables within articles themselves.
- footnotes within the tables that are to be written under the table, rather than as endnotes are not working well.

Part of the problem here is that JATS allows for inaccessible tables to be specified, thus user content will not always produce accessible tables. The row/column headings should be specified in the JATS and then picked up so if it is not correct there, it won't be correct on the page either.

Therefore we should - 
- [x] ensure non-article (i.e. non user content) tables are accessible.  Each data cell is expected to be understood in the context of one column header and one row header - so a single row of headers and a single column of headers for each table, using the `<th>` tag.
- [x] add user content/jats issue to Phase 2 of A11y project.

**Comment by StephDriver** [comment]
from Screen reader testing: 
- Sample 2, item 66. dates are given in a table. This has no column header. I suggest "Date" as the header, or not to use tables here.
- Sample 4, item 33. table does not have unique headers for rows and columns, and what row headers exist, are only shown visually with `<strong>` tags rather than `<th>`
- Sample 4, item 35. following footnotes inside the table (e.g. in table 7) link to footnotes by moving the page on the screen, but reading or tabbing onwards takes you back to the original location, so footnote cannot be read.
- Sample 4, item 36. table footnotes do not have the letter of the footnote to which they relate.
- Sample 4, item 37. table footnotes are in paragraphs, rather than as a list.

**Comment by StephDriver** [comment]
https://jats4r.niso.org/accessibility/
JATS table a11y

**Comment by StephDriver** [comment]
Process:

- [ ] find tables and add comment above each.
- [ ] first review - expand comment to note obvious issues, and if none, which type of table (see below)
- [ ] fix obvious issues, and update comment to type of table.
- [ ] for each type of table, review in more detail, and fix any a11y markup required.
- [ ] Review all as to whether captions and summaries are required (probably not for our use, but check).
- [ ] remove the comments

Types of table (as per [WCAG guidelines](https://www.w3.org/WAI/tutorials/tables/)):
- V1 - One header (either row or column)
- V2 - header row and column row
- V3 - Irregular headers
- V4 - Multilevel headers

**Comment by StephDriver** [comment]
Working on tables within `themes` directory only, and using [W3C Tables Tutorial](https://www.w3.org/WAI/tutorials/tables/) for guidance.

**Comment by StephDriver** [comment]
templates outside the clean theme that include tables (to investigate):
(see https://github.com/openlibhums/janeway/issues/4215#issuecomment-2483302101 for how this list was generated)

- [x] common/accounts/register_privacy_policy.html
- [x] common/elements/altmetric_badges.html
- [x] common/elements/analytics.html
- [ ] common/elements/edit_profile_css_ block.html
- [ ] common/elements/edit_profile_js_block.html
- [ ] common/elements/journal/editorial_team_country.html
- [ ] common/elements/journal/editorial_team_image.html
- [ ] common/elements/orcid_registration.html
- [ ] common/elements/password_rules.html
- [ ] common/elements/profile_image.html
- [ ] common/elements/skip_to_main_content.html
- [ ] common/elements/static-error-messages.html
- [ ] element.template_path
- [ ] elements/article_meta_tags.html
- [ ] elements/forms/errors.html
- [ ] elements/funder_info_for_readers.html
- [ ] elements/journal/issue_block.html
- [ ] elements/journal/issue_top.html
- [ ] elements/social_meta.html

**Comment by StephDriver** [comment]
> templates outside the clean theme that include tables (to investigate)

👉 no tables found in templates included from outside the theme directory..
